tests/plugins.py

```diff
@@ -0,0 +1,24 @@
+"""Plugins for testing purposes.
```

Comment:
Added this file so the test can use the default discover behaviour, leading to the issue explained above (File "<string>" instead of the actual file name). Having these tests in the test file itself will not trigger that issue.

Reply:
Ok, that looks fine.